### PR TITLE
Implemented 'Print stack of client-side JS errors' (close #2043)

### LINKED ISSUE #2043: Print stack of client-side JS errors
### Are you requesting a feature or reporting a bug?
Feature (from https://testcafe-discuss.devexpress.com/t/718)

### What is the current behavior?
TestCafe shows only the error's message.

### What is the expected behavior?
TestCafe should show full error information.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js

```

### Specify your

* operating system:
* testcafe version:
* node.js version:

### PR DESCRIPTION
**Pull request by miherlosev**
@AndreyBelym @kirovboris 

Review TestCafe related part.
Second part of feature in this [PR](https://github.com/DevExpress/testcafe-hammerhead/pull/1735) and it will be reviewed separately.

### CODE REVIEW

File: test/functional/fixtures/run-options/print-stack-js-errors/test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const expect = require('chai').expect;
+
+describe('Print stack of client-side JS errors', () => {
+    it('Basic', () => {
+        return runTests('testcafe-fixtures/print-stack-js-errors-test.js', 'Basic', {
+            shouldFail:         true,
+            only:               'chrome',
+            printStackJsErrors: true
```
Comment (helen-dikareva) [inline test/functional/fixtures/run-options/print-stack-js-errors/test.js]: Now we can get rid of `printStackJsErrors` setting

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
ping @kirovboris @helen-dikareva

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Pull request merged**